LICENSE

```diff
@@ -178,15 +179,15 @@ Apache License
    APPENDIX: How to apply the Apache License to your work.
 
       To apply the Apache License to your work, attach the following
-      boilerplate notice, with the fields enclosed by brackets "{}"
+      boilerplate notice, with the fields enclosed by brackets "[]"
       replaced with your own identifying information. (Don't include
       the brackets!)  The text should be enclosed in the appropriate
       comment syntax for the file format. We also recommend that a
       file or class name and description of purpose be included on the
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-    Copyright 2015 HM Revenue & Customs
+   Copyright [yyyy] [name of copyright owner]
```

Comment:
Can we add "2020 HM Revenue & Customs" here please.

Reply:
The licence text needs to exactly match the template at: https://www.apache.org/licenses/LICENSE-2.0.txt to satisfy `sbt-auto-build`. (the template indicates how to apply the licence to the source files, and shouldn't be filled in itself)